sentry-ruby/spec/sentry/breadcrumb/redis_logger_spec.rb

```diff
@@ -0,0 +1,81 @@
+require "spec_helper"
+require "fakeredis"
```

Comment:
I think it's ok to put this in `spec_helper` and before `require "sentry-ruby"`. Then we don't need to force the `redis.rb` reload anymore. Wdyt?

Reply:
Good idea!